app/conversations/routes.py

```diff
@@ -145,3 +144,23 @@ def get_conversations():
     response = {"conversations": results}
 
     return jsonify(response), 200
+
+
+@bp.route("/conversation/<conversation_uuid>", methods=["GET"])
+@cross_origin()
+def get_conversation(conversation_uuid):
+    """
+    Get a single conversation.
```

Comment:
This needs a parameters section in the description and a description of the response.

Reply:
Ok. Got lazy as we write such self-documenting code.